calico/common.py

```diff
@@ -446,6 +460,9 @@ def validate_rules(profile_id, rules):
                     continue
 
                 if ports is not None:
+                    if protocol not in ("tcp", "udp"):
+                        issues.append("%s is only valid with tcp or udp in "
```

Comment:
According to iptables itself, port is valid for tcp, udp, udplite, sctp and dccp, not just tcp and udp.